dll/win32/comctl32/static.c

```diff
@@ -626,7 +626,15 @@ static void STATIC_PaintTextfn( HWND hwnd, HDC hdc, DWORD style )
     }
     else
     {
-        DrawTextW( hdc, text, -1, &rc, format );
+        UINT flags = DST_TEXT | ((style & WS_DISABLED) ? DSS_DISABLED : DSS_NORMAL);
+        if (format & DT_HIDEPREFIX)
+            flags |= DSS_HIDEPREFIX;
+        hBrush = GetSysColorBrush(COLOR_GRAYTEXT);
+        DrawStateW(hdc, hBrush, NULL,
+                   (LPARAM)text, (WPARAM)wcslen(text),
```

Comment:
```suggestion
        DrawStateW(hdc, hBrush, STATIC_DrawTextCallback,
                   (LPARAM)text, (WPARAM)format,
```
We pass the `format` flags previously calculated to the callback.